integration-tests/src/test/kotlin/com/google/devtools/ksp/test/PlaygroundIT.kt

```diff
@@ -0,0 +1,30 @@
+package com.google.devtools.ksp.test
+
+import TestsCommon
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.Assert
+import org.junit.Test
+import java.io.File
+import java.util.jar.*
+
+class PlaygroundIT : TestsCommon() {
+    override val testProjectName = "playground"
+
+    @Test
+    fun testPlayground() {
+        val gradleRunner = GradleRunner.create().withProjectDir(testProjectDir)
```

Comment:
I haven't used `GradleRunner` before. Does each test JVM fork starts its own Gradle daemon?

Reply:
According to the doc, yes, although I didn't verify it.
https://docs.gradle.org/current/userguide/test_kit.html#sec:controlling_the_build_environment

An observation from another perspective is that it's a bit slow to start. For integration tests that involves Gradle, I guess I can accept it, though.